bytes.js

```diff
@@ -0,0 +1,47 @@
+const toHex = d => d.reduce((hex, byte) => hex + byte.toString(16).padStart(2, '0'), '')
+const fromHex = hex => {
+  if (!hex.length) return new Uint8Array(0)
+  return new Uint8Array(hex.match(/.{1,2}/g).map(b => parseInt(b, 16)))
```

Comment:
I think just `/../` should be OK here

Reply:
my regex-fu is not good enough to know which is better or worse here. i bet @ribasushi knows, perl hackers know everything about regex 😝

Reply:
Um...
`/.{1,2}/` == `one or two bytes (any byte value)`
`/../` == `exactly two bytes (any byte value)`

I think @rvagg has a point, as you already did `toString(16).padStart(2, '0')`